test/integration/lib/tx-list-items.js

```diff
@@ -16,7 +16,7 @@ QUnit.test('renders list items successfully', (assert) => {
 
 global.ethQuery = global.ethQuery || {}
 global.ethQuery.getTransactionCount = (_, cb) => {
-  cb(null, '0x3')
+  cb(null, '0x0')
```

Comment:
Any reason that this mock return nonce, and the nonce in the json test data, is changed?

Reply:
The tx with the nonce '0x3' has a failed status, whereas there's a tx with a nonce of '0x0' in the pending tx list. I changed `getTransactionCount` here so that the retry button would appear in that '0x0' tx. I actually didn't need to change the `nonce` in the json test data so I'll revert that.